Calling the FeeHistory RPC during startup will cause a panic

#### System information

Geth version: `v0.5.1`
OS & Version: Windows/Linux/OSX
Commit hash : ced1753d5026fddc8ec95d86e452d97e65c38fd7

#### Expected behaviour
Normal startup

#### Actual behaviour
Panic

#### Steps to reproduce the behaviour
Calling the FeeHistory RPC during startup may trigger it.

#### Backtrace

````
INFO [11-11|09:44:18.161] Started log indexer
INFO [11-11|09:44:18.162] Loaded local transaction journal         transactions=2 dropped=0
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x102db2d]

goroutine 8571 [running]:
math/big.(*Int).Set(...)
 math/big/int.go:97
github.com/ethereum/go-ethereum/eth/gasprice.(*Oracle).suggestTipCapInternal(0xc0003580b0, {0x2628b18, 0xc0036a2230}, 0xc000325408)
 github.com/ethereum/go-ethereum/eth/gasprice/gasprice.go:235 +0xe8d
github.com/ethereum/go-ethereum/eth/gasprice.(*Oracle).processBlock(0xc0003580b0, {0x2628b18, 0xc0036a2230}, 0xc000a10510, {0xc000a9b290, 0x2, 0x0?})
 github.com/ethereum/go-ethereum/eth/gasprice/feehistory.go:92 +0x1b0
github.com/ethereum/go-ethereum/eth/gasprice.(*Oracle).FeeHistory.func1()
 github.com/ethereum/go-ethereum/eth/gasprice/feehistory.go:320 +0x695
created by github.com/ethereum/go-ethereum/eth/gasprice.(*Oracle).FeeHistory in goroutine 8567
 github.com/ethereum/go-ethereum/eth/gasprice/feehistory.go:289 +0x82c

````

--- a/eth/gasprice/gasprice.go
+++ b/eth/gasprice/gasprice.go
@@ -148,6 +148,9 @@ func NewOracle(backend OracleBackend, params Config, startPrice *big.Int) *Oracl
 	return &Oracle{
 		backend:          backend,
 		lastPrice:        startPrice,
+		lastBaseFee:      new(big.Int),
+		lastMinGasTipCap: new(big.Int),
+		lastEnvelopeFee:  new(big.Int),
 		maxPrice:         maxPrice,
 		ignorePrice:      ignorePrice,
 		checkBlocks:      blocks,

